src/components/ebay-radio/index.marko

```diff
@@ -11,7 +11,8 @@ $ input.toJSON = noop;
         type="radio"
         class="radio__control"
         onClick("handleClick")
-        onFocus("handleFocus")/>
+        onFocus("handleFocus")
+        on-keydown("handleKeyDown")/>
```

Comment:
Follow the other syntax
`onKeydown` 
Also for the methodname, do `handleKeydown` to match the text. (Don't worry about changing it in the 5.0 branch)